examples/constant_memory.c

```diff
@@ -18,7 +18,9 @@ int main() {
     /* Set the worksheet options. */
     lxw_workbook_options options = {.constant_memory = LXW_TRUE,
                                     .tmpdir = NULL,
-                                    .use_zip64 = LXW_FALSE};
+                                    .use_zip64 = LXW_FALSE,
+                                    .output_buffer = NULL,
```

Comment:
Move the `output_buffer` variables after `.constant_memory` in struct to indicate the relative importance of the features. It will break ABI either way.

Reply:
I assume you mean in `workbook.h`. I agree that it should be higher, but I did try to preserve the API at least, specifically when initializing the struct without using named fields (like in some of the tests).